Packs/Salesforce/Integrations/Salesforce/Salesforce.py

```diff
@@ -0,0 +1,1037 @@
+    remote_args = GetModifiedRemoteDataArgs(args)
+    last_update = remote_args.last_update
+    last_update_sfdc = parse(last_update).isoformat().split("+")[0].split(".")[0] + "Z"
+    demisto.debug(f'SalesforcePy : * START * Performing get-modified-remote-data command. Last update is: {last_update}')
+
+    cases = client.queryObjects(['Id'], 'Case', f"LastModifiedDate >= {last_update_sfdc}").get('records')
+    for item in cases:
+        modified_records_ids.append(item['Id'])
+
+    demisto.debug(
+        f"SalesforcePy : * END * Performing get-modified-remote-data command. Results: {','.join(modified_records_ids)}")
+
+    return GetModifiedRemoteDataResponse(modified_records_ids)
+
+
+def get_remote_data_command(client, args):
+    parsed_args = GetRemoteDataArgs(args)
+    new_incident_data = {}
+    entries = []
+    try:
+        new_incident_data, case_comments = get_data(client, parsed_args.remote_incident_id, parsed_args.last_update)
+        new_incident_data['id'] = parsed_args.remote_incident_id
+        new_incident_data['in_mirror_error'] = ''
+
+        if len(case_comments) > 0:
+            for cc in case_comments:
+                owner = cc.get('Owner', '')
+                entries.append({
+                    'Type': EntryType.NOTE,
+                    'Contents': f"# Case Comment From Salesforce on {cc.get('LastModifiedDate', '')}\n*Created By*: {owner.get('FirstName', '')} {owner.get('LastName', '')} ({owner.get('Username', '')})\n\n{cc.get('CommentBody', '')}",
+                    'ContentsFormat': EntryFormat.MARKDOWN,
+                    'Tags': ['Salesforce Case Comment'],  # the list of tags to add to the entry
+                    'Note': True,
+                    'IgnoreAutoExtract': True
+                })
+
+        # close xsoar incident when SFDC case is closed
+        if new_incident_data.get('Status') == 'Closed':
+            if demisto.params().get('close_incident'):
+                demisto.debug(f'case is closed: {new_incident_data}')
+                entries.append({
+                    'Type': EntryType.NOTE,
+                    'Contents': {
+                        'dbotIncidentClose': True,
+                        'closeReason': f'Case closed in Salesforce on {new_incident_data.get("LastModifiedDate")}'
+                    },
+                    'ContentsFormat': EntryFormat.JSON
+                })
+
+        return GetRemoteDataResponse(mirrored_object=new_incident_data, entries=entries)
+
+    except Exception as e:
+
+        if new_incident_data:
+            new_incident_data['in_mirror_error'] = str(e)
+        else:
+            new_incident_data = {
+                'id': parsed_args.remote_incident_id,
+                'in_mirror_error': str(e)
+            }
+        return GetRemoteDataResponse(
+            mirrored_object=new_incident_data,
+            entries=[]
+        )
+
+
+def fetchIncident(client):
+    fetchType = demisto.params().get("fetchType")
```

Comment:
put demisto.params() into a variable and then use in the rest of the function

Reply:
fixed